src/collectors/postgres/postgres.py

```diff
@@ -54,9 +76,9 @@ def get_default_config(self):
         Return default config.
         """
         config = super(PostgresqlCollector, self).get_default_config()
+
         config.update({
             'path': 'postgres',
-            'host': 'localhost',
```

Comment:
Currently this PR is a breaking change.  However, if you keep this config option, add it to the instance list in the `collect` method if it isn't already in it, and then exclude `host` from falling back to the default in `_get_config`, then it would not be a breaking change.